src/components/viewmodels/roomlist/RoomListHeaderViewModel.tsx

```diff
@@ -128,8 +128,8 @@ export function useRoomListHeaderViewModel(): RoomListHeaderViewState {
     const isSpaceRoom = Boolean(activeSpace);
 
     const canCreateRoom = hasCreateRoomRights(matrixClient, activeSpace);
-    const canCreateVideoRoom = useFeatureEnabled("feature_video_rooms");
```

Comment:
When playing with the video room, I saw this mistake. We didn't checked the room creation rights for video room.